response_operations_ui/templates/collection_exercise/ci-versions.html

```diff
@@ -38,7 +39,9 @@ <h1>Choose CIR version for EQ formtype {{ form_type }}</h1>
                               "text": "Version " + ci.ci_version|string,
                               "description": "Published: " + ci.published_at,
                           },
-                          "name": ci.ci_version|string
+                          "name": ci.ci_version|string,
```

Comment:
This should probably be renamed to reflect it's the registry_instrument `guid` and not the version.